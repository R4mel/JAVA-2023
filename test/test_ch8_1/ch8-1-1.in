1
17
6
4
4
4
6
1
0
0
0
