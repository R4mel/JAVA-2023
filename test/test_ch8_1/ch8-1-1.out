
************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 17

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 4
[0] t0.txt               F 2023-11-15 07:13        51
[1] t1.txt               F 2023-11-15 07:13       102
[2] t2.txt               F 2023-11-15 07:13       153
[3] t3.txt               F 2023-11-15 07:13       204
-----------------
[data] directory: 4 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 4
[0] t0.txt               F 2023-11-15 07:13        51
[1] t1.txt               F 2023-11-15 07:13       102
[2] t2.txt               F 2023-11-15 07:13       153
[3] t3.txt               F 2023-11-15 07:13       204
[4] t4.txt               F 2023-11-15 07:13       255
[5] t5.txt               F 2023-11-15 07:13       306
[6] t6.txt               F 2023-11-15 07:13       357
[7] t7.txt               F 2023-11-15 07:13       408
-----------------
[data] directory: 8 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 4
[0] t0.txt               F 2023-11-15 07:13        51
[1] t1.txt               F 2023-11-15 07:13       102
[2] t10.txt              F 2023-11-15 07:13       561
[3] t11.txt              F 2023-11-15 07:13       612
[4] t2.txt               F 2023-11-15 07:13       153
[5] t3.txt               F 2023-11-15 07:13       204
[6] t4.txt               F 2023-11-15 07:13       255
[7] t5.txt               F 2023-11-15 07:13       306
[8] t6.txt               F 2023-11-15 07:13       357
[9] t7.txt               F 2023-11-15 07:13       408
[10] t8.txt               F 2023-11-15 07:13       459
[11] t9.txt               F 2023-11-15 07:13       510
-----------------
[data] directory: 12 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 1
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll                                                   =
=================================================================
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 0

************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 0

Good bye!!
